### PR TITLE
feat: Adds initial env.example file.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: env.example [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+JITSI_ASAP_JWT_ACCEPTED_HOOK_ISS=
+
+#
+#KEY_PREFIX_PATTERN='^(.*)/(.*)$'
+
+#
+#SIP_ADDRESS_PATTERN='(^sips?:)?(.*)(@.*)'
+
+#
+#SIP_JIBRI_INBOUND_EMAIL='inbound-sip-jibri@jitsi.net'
```
Comment (damencho) [inline env.example]: Do we want to leave this as the default value? Or we can leave it empty and skip functionality if missing?

File: env.example [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+#
+#SIP_JIBRI_OUTBOUND_EMAIL='outbound-sip-jibri@jitsi.net'
```
Comment (damencho) [inline env.example]: Same and for this one?

File: env.example (hunk 1 above)
Comment (raluca8x8) [reply env.example]: We can leave it empty

File: env.example (hunk 2 above)
Comment (raluca8x8) [reply env.example]: Same

**Review by raluca8x8 — APPROVED** [review]

File: src/config/config.ts [hunk 3]
```diff
@@ -31,21 +31,21 @@ const env = envalid.cleanEnv(process.env, {
     CANDIDATE_TTL_SEC: envalid.num({ default: 120 }),
     SESSION_TTL_SEC: envalid.num({ default: 86400 }),
     ASAP_PUB_KEY_TTL: envalid.num({ default: 3600 }),
-    SYSTEM_ASAP_BASE_URL_MAPPINGS: envalid.json({ example: '[{"kid": "kidPattern", baseUrl": "https://asap.org"}]' }),
-    SYSTEM_ASAP_JWT_AUD: envalid.str(),
-    SYSTEM_ASAP_JWT_ACCEPTED_HOOK_ISS: envalid.str(),
-    JITSI_ASAP_BASE_URL_MAPPINGS: envalid.json({ example: '[{"kid":"kidPattern","baseUrl":"https://asap.org"]' }),
-    JITSI_ASAP_JWT_AUD: envalid.str(),
-    JITSI_ASAP_JWT_ACCEPTED_HOOK_ISS: envalid.str(),
+    SYSTEM_ASAP_BASE_URL_MAPPINGS: envalid.json({ default: '[{"kid": "^jitsi/(.*)$", "baseUrl": "https://localhost:18443/server"}]' }),
```
Comment (aaronkvanmeerten) [inline src/config/config.ts]:
```suggestion
    SYSTEM_ASAP_BASE_URL_MAPPINGS: envalid.json({
        default: [ { 'kid': '^jitsi/(.*)$',
            'baseUrl': 'https://localhost:18443/server' } ]
     }),
```
Default needs to be an object, not a string

File: src/config/config.ts [hunk 4]
```diff
@@ -31,21 +31,21 @@ const env = envalid.cleanEnv(process.env, {
...
+    SYSTEM_ASAP_JWT_AUD: envalid.str({ default: 'jitsi-component-selector' }),
+    SYSTEM_ASAP_JWT_ACCEPTED_HOOK_ISS: envalid.str({ default: 'jitsi-component-sidecar,jicofo' }),
+    JITSI_ASAP_BASE_URL_MAPPINGS: envalid.json({ default: '[{"kid": "^jitsi/(.*)$", "baseUrl": "https://localhost:18433/clients"} ]' }),
```
Comment (aaronkvanmeerten) [inline src/config/config.ts]:
```suggestion
    JITSI_ASAP_BASE_URL_MAPPINGS: envalid.json(
        { default: [ { 'kid': '^jitsi/(.*)$',
             'baseUrl': 'https://localhost:18433/clients' } ]
     }),
```
Same as the above for this one

**Review by aaronkvanmeerten — CHANGES_REQUESTED** [review]
envalid.json needs to be a valid object

**Review by aaronkvanmeerten — APPROVED** [review]

**Pull request merged**